### DISCUSSION TITLE
Nested collections & Permissions

### DISCUSSION THREAD

**Discussion by Siem2l** [outline/outline#2133] [Ideas]
Currently collections are the only files that have permissions. It would be a nice additions to have nested collections to sub categorize for a better overview. With the permissions this would also make it possible to have a "category" with only a few of the sub categories visible.

for example
1.Assignment X (Visible for: Group X, Group Y, Teaching assistant)
> Assignment evaluation (visible for: Teaching assistant)
> Group X (Visible for : Group X, Teaching assistant)
> Group Y(Visible for: Group Y, Teaching assistant)
2. Assignment Y
etc..

**Comment by Gredin67** [comment]
@tommoor is there a plan on implementing this "wiki"-like fine grain permissions functionality?

**Reply by tommoor** [reply]
It's not currently on the roadmap, it becomes very hard to reason about who has access to what with nested permissions – it also necessarily impacts performance of the app. Instead we'll have individual document sharing separate to the collection structure
